Program.cs

```diff
@@ -0,0 +1,207 @@
+﻿
+using Refrigerator_EX;
+using System;
+
+class Program
+{
+    public static List<Refrigerator> SortRefrigeratorsByFreeSpace(List<Refrigerator> refrigerators)
+    {
+        List<Refrigerator> sortRefrigerators = refrigerators.OrderByDescending(refrigerator => refrigerator.spaceInRefrigerator()).ToList();
```

Comment:
`sortRefrigerators` is a name of a function, you should call the variable `sortedRefrigerators`